public/js/widget_config.js

```diff
@@ -251,13 +296,13 @@ const initWidgetConfig = function (objSocket) { // eslint-disable-line no-unused
 
   jQuery('#newWidget #newWidgetType').selectmenu({
     change: function (event, ui) {
-      const widgetType = ui.item.value
-      jQuery('#newWidget .newWidgetType').hide()
-      jQuery(`#newWidget #${widgetType}`).show()
+      setNewWidgetDialogType(ui.item.value)
```

Comment:
I decided to create this function, because simply triggering the 'change' event on the jQuery widget wouldn't run the function.